### PR TITLE
applications: nrf5340_audio: Rename content and renderer files

### PR DESCRIPTION
**Pull request by alexsven**
- Create subfolder media under bt_content_ctrl
- Create subfolder volume under bt_rend
- Make header files of submodules internal

### CODE REVIEW

File: applications/nrf5340_audio/src/bluetooth/bt_content_control/media/bt_content_ctrl_media_internal.h [hunk 1]
```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#ifndef _BT_MEDIA_CONTROL_H_
-#define _BT_MEDIA_CONTROL_H_
+#ifndef _BT_CONTENT_CTRL_MEDIA_H_
```
Comment (erikrobstad) [inline applications/nrf5340_audio/src/bluetooth/bt_content_control/media/bt_content_ctrl_media_internal.h]:
```suggestion
#ifndef _BT_CONTENT_CTRL_MEDIA_INTERNAL_H_
```

I need to do the same, hehe

File: applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * @return	0 for success, error otherwise.
+ */
+int bt_rend_vol_discover(struct bt_conn *conn);
+
+/**
+ * @brief	Initialize the Volume Control Service client.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int bt_rend_vol_ctlr_init(void);
```
Comment (erikrobstad) [inline applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h:77]: I remember we discussed this, but I don't remember the conclusion.
'ctlr' is correct to use here?

File: applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h [hunk 3]
```diff
@@ -42,14 +42,14 @@ int bt_rend_volume_set(uint8_t volume, bool from_vcp);
  *
  * @return	0 if success, error otherwise
  */
-int bt_rend_mute(bool from_vcp);
+int bt_rend_volume_mute(bool from_vcp);
```
Comment (erikrobstad) [inline applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h:45]: Use 'vol' here?

File: applications/nrf5340_audio/src/bluetooth/Kconfig [hunk 4]
```diff
@@ -570,20 +570,20 @@ module = BT_MGMT
...
+module-str = bt_content_ctrl
 source "subsys/logging/Kconfig.template.log_config"
 
-module = BT_MEDIA_CONTROL
-module-str = bt_media_control
+module = BT_CONTENT_CTRL_MEDIA
+module-str = bt_content_ctrl_media
 source "subsys/logging/Kconfig.template.log_config"
 
 module = BT_RENDERER
```
Comment (erikrobstad) [inline applications/nrf5340_audio/src/bluetooth/Kconfig]:
```suggestion
module = BT_REND
```

**Review by erikrobstad — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h (hunk 2 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h:77]: yes, this is the role volume controller

File: applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h (hunk 3 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_rend.h:45]: No, the vol is in the file bt_rend_vol.h/c

File: applications/nrf5340_audio/src/bluetooth/Kconfig (hunk 4 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/Kconfig]: Done

File: applications/nrf5340_audio/src/bluetooth/bt_content_control/media/bt_content_ctrl_media_internal.h (hunk 1 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_content_control/media/bt_content_ctrl_media_internal.h]: Done

**Review by erikrobstad — APPROVED** [review]

**Pull request merged**